app/src/main/java/com/example/yard/Activities/LoginActivity.java

```diff
@@ -23,19 +23,19 @@ protected void onCreate(Bundle savedInstanceState) {
       goMainActivity();
     }
     tvEmailAddress = findViewById(R.id.tvEmailAddress);
-    password = findViewById(R.id.etPassword);
-    TextView signUp = findViewById(R.id.tvSign);
-    Button logInButton = findViewById(R.id.btnLogin);
-    signUp.setOnClickListener(
+    tvPassword = findViewById(R.id.etPassword);
```

Comment:
The renaming is in another PR.

Also this is modifying the files under `Activities` folder instead of `activities`. Please make sure you're modifying the correct files.